R/calculate-grades.R

```diff
@@ -197,15 +197,23 @@ raw_over_max <- function(grades_mat, assignments){
 #' 
 #' @export
 equally_weighted <- function(grades_mat, category, assignments, weights = c()){
-  grades_mat[,category] <- rowMeans(grades_mat[, assignments], na.rm = TRUE)
+  if (length(assignments) == 1){
```

Comment:
Would it also be wise to check if the length is non-positive? Ideally, this would not happen but it may be plausible if someone interacts oddly with our policy file or the package.